starter-project-web-vue2/components/yared/Blog.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+<div class="blog">
+      <v-card
+    :to="'/yared/blogs/' + blog._id"
+    class="mx-auto"
+    max-width="400"
+    nuxt
+  >
+    <v-img
+      class="white--text align-end"
+      height="200px"
+      src="https://cdn.vuetifyjs.com/images/cards/docks.jpg"
+    >
+      <v-card-title>{{blog.title}}</v-card-title>
+    </v-img>
+
+    <v-card-subtitle class="pb-0">
+      ID: {{blog._id}}
+    </v-card-subtitle>
+   <v-card-actions>
+      <v-btn
+        color="yellow"
+      >
+        Update
+      </v-btn>
+      <v-btn
+      style="position:absolute; right:0%;"
+      class="mx-2"
+      fab
+      dark
+      small
+      color="red"
+    >
+      <v-icon dark>
+        mdi-minus
+      </v-icon>
+    </v-btn>
+    </v-card-actions>
+  </v-card>
+</div>
+    
+</template>
+
+<script>
+export default {
+    name: "Blog",
+    props:{
+        blog: Object,
+    }
+    
+}
+</script>
+
+<style scoped>
+.blog{
```

Comment:
we could use vuetify classes here like wrapping in _v-col_ and giving it _cols=12_ and use _ma-2_  for the margin (refer [here](https://vuetifyjs.com/en/styles/spacing/#how-it-works) for more info